.travis.yml

```diff
@@ -1,29 +1,37 @@
 language: php
+sudo: false
 
 php:
   - 5.6
+  - 7.0
+  - 7.1
 
 env:
-  - MONGO_DRIVER=mongo
+  global:
+    - DRIVER_VERSION="stable"
+    - ADAPTER_VERSION="^1.0.0"
 
 matrix:
   include:
     - php: 5.6
-      env: MONGO_DRIVER=mongo COMPOSER_FLAGS='--prefer-lowest --prefer-stable'
-    - php: 7.0
-      env: ADAPTER_VERSION="^1.0.0" MONGO_DRIVER=mongodb
-
-sudo: false
+      env: DRIVER_VERSION="1.5.8" PREFER_LOWEST="--prefer-lowest"
```

Comment:
naming it ``COMPOSER_FLAGS`` is better than ``PREFER_LOWEST``, as it allows to pass any flags to the composer command